### DISCUSSION TITLE
brew install fails in GitPod

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [Homebrew/discussions#813] [Linux]
Hello everyone! I'm learning to use GitPod now and have [this](https://gitlab.com/dvfu-university/calculus/octave-in-gitpod-example) repo with [`.gitpod.yml`](https://www.gitpod.io/docs/config-gitpod-file/):
```yml
tasks:
  - init: brew install octave
    command: chmod +x main.m
```
When I create new workspace for aforementioned repo I get the following error as the installation result:
```
Project ERROR: Could not resolve SDK Path for 'macosx' using --show-sdk-path
```
<img width="1792" alt="Screen Shot 2021-02-15 at 7 45 57 PM" src="https://user-images.githubusercontent.com/42812113/107930256-79a50e00-6fc6-11eb-9e37-8a1cc32ab962.png">

How to avoid this problem?

**Comment by SMillerDev** [comment]
This is a question for gitpod, but I see that they're installing on Linux so there's a good chance that software just isn't supported on linux.

**Reply by EmilyGraceSeville7cf** [reply]
Thanks for a response! I wrote them about it.

**Reply by danielnachun** [reply]
Are they using an outdated repo for Homebrew?  I fixed that issue a while ago and `octave` and all its dependencies are bottled.  Either it's outdated or they managed to use homebrew/core instead of linuxbrew/core.
